packages/@react-native-windows/cli/src/runWindows/runWindowsOptions.ts

```diff
@@ -128,6 +130,11 @@ export const runWindowsOptions: CommandOption[] = [
     description: 'Do not deploy the app',
     default: false,
   },
+  {
+    name: '--deploy-from-layout',
```

Comment:
add the option to the cli readme.md 
we'll also want to update the samples repo with this option

Reply:
Updated the CLI README. From what I can find, we have flags for react-native-windows-init in the samples repo, but not the CLI.